### ISSUE TITLE
JAWS 2024 Ver. 2024.2405.83 - Doesn't navigate correctly and announces BLANK

### ISSUE THREAD

**Issue by rpilman** [FreedomScientific/standards-support#843]
### Summary
Using JAWS on our published output worked before updating to JAWS 2024 Ver. 2024.2405.83
Now, after launching the course and using the arrow key to navigate the course, JAWS just announces BLANK. 

Trying to navigate the course with JAWS 2024.2405.83 doesn't work and announces blank 

1. Download the sample published output from Storyline 360
[SL360-7421-SupportTest_Screen Readers_SL360.zip](https://github.com/user-attachments/files/15595625/SL360-7421-SupportTest_Screen.Readers_SL360.zip)
2. Open JAWS 2024.2405.83
3. Open the story.html 
4. this will load the course 
5. press the down arrow key to navigate 

### Expected result

I would expect JAWS to start reading through objects on the slide and then player controls 

### Actual result

JAWS 2024 just announces BLANK every time the down arrow is pressed.

Notes:
I tested with JAWS 2023 and it works fine. 

#### JAWS version and build number
- JAWS 2024 Ver. 2024.2405.83

 
#### Operating System and version
- Windows 11 Pro
- Version	10.0.22631 Build 22631

#### Browser and version:
- Chrome Version 125.0.6422.141 (Official Build) (64-bit)

**Comment by rpilman** [comment]
I tested many different other websites like capitalone.com  Deque.com and seeing the same problem there.  PRessing the down arrow key says blank.

**Comment by stevefaulkner** [comment]
tested provided test case in JAWS 2024 Ver. 2024.2405.48 (latest public release) Could not reproduce issue
